package.json

```diff
@@ -47,6 +47,7 @@
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
     "ripple-address-codec": "^5.0.0-beta.1",
```

Comment:
This should probably be bumped to a non beta too.

Reply:
dependabot did not seem to catch that it seems

Reply:
I bumped to `5.0.0`